lib/mixlib/install/generator/powershell/scripts/helpers.ps1.erb

```diff
@@ -110,20 +118,25 @@ function Get-WebContentOnCore {
 function Test-ProjectPackage {
   [cmdletbinding()]
   param ($Path, $Algorithm = 'SHA256', $Hash)
+  if (!$env:Valid_ProjectPackage){
+    Write-Verbose "Testing the $Algorithm hash for $path."
+    $ActualHash = if (Is-FIPS) {
+      (Custom-GetFileHash -Algorithm $Algorithm -Path $Path).Hash.ToLower()
+    } else {
+      (Get-FileHash -Algorithm $Algorithm -Path $Path).Hash.ToLower()
```

Comment:
ahh. right. be aware that cmdlet was introduced in ps v4 and will therefore fail on OS versions prior to 2012R2 unless their ps has been updated.